File: APIWeb/APIWeb/Model/Domain/Order.cs
namespace APIWeb.Model.Domain
{
    public class Order
    {
        public Guid Id { get; set; }
        public DateTime OrderTime { get; set; }
        public DateTime? AcceptTime { get; set; }
        public Guid CustomerID { get; set; }
        public string DeliveryProvince { get; set; }
        public string DeliveryAddress { get; set; }
        public Guid? EmployeeID { get; set; }
        public Guid? ShipperID { get; set; }
        public DateTime? ShippedTime { get; set; }
        public DateTime? FinishedTime { get; set; }
        public int StatusID { get; set; }

        public Employees Employee { get; set; }
        public Shippers Shipper { get; set; }
        public Customers Customer { get; set; }
        public StatusOrder Status { get; set; }


    }
}

File: API/REACT/client-app/src/app/models/order.ts
import { DetailOrder } from "./detailorder"

export type Order = {
    id: string
    orderTime: string
    acceptTime: string
    deliveryProvince: string
    deliveryAddress: string
    shipperID: string
    shippedTime: string
    finishedTime: string
    statusID: number
    customerID: string
    employeeID: string
    detailOrder: DetailOrder[]
}
